python/src/pywy/basic/model/ops.py

```diff
@@ -221,6 +221,13 @@ def get_dType(self):
     def inputs_required(self):
         return 1
 
+class LogisticRegression(Op):
```

Comment:
Not sure that's the right place for this. Checkout this package: 
from pywy.basic.model.models